src/lightkurve/io/eleanorlite.py

```diff
@@ -0,0 +1,114 @@
+"""Reader for GSFC-ELEANOR-LITE light curve files. 
+Details can be found at https://archive.stsci.edu/hlsp/eleanor and https://archive.stsci.edu/hlsp/gsfc-eleanor-lite
+"""
+from ..lightcurve import TessLightCurve
+from ..utils import TessQualityFlags
+from astropy import units as u
+
+from .generic import read_generic_lightcurve
+
+import numpy as np
+
+def read_eleanorlite_lightcurve(filename,
+    time_column="TIME",
+    flux_column="CORR_FLUX", 
+    flux_err_column="FLUX_ERR", 
+    cadenceno_column="FFIINDEX",
+    centroid_col_column="X_CENTROID",
+    centroid_row_column="Y_CENTROID",
+    quality_column="QUALITY",
+    quality_bitmask="default", 
+    **kwargs):
+    """Returns a `TessLightCurve` object given a light curve file from the GSFC Eleanor-lite Pipeline.
+
+    By default, eleanor's `CORR_FLUX` column is used to populate the `flux` values. Note that the "FLUX_ERR" column in the 
+    Eleanor FITS file is referred to the uncertainty of "RAW_FLUX", not "CORR_FLUX". 
+    Thus the uncertainty reported in the 'flux_err' column here is calculated as follows: corr_flux_err = corr_flux*raw_flux_err/raw_flux. 
+    For completeness, the original raw_flux's error is added as a "raw_flux_err" column.
+
```

Comment:
Add explanation of quality bits, a la @ethankruse comments from #1217

Reply:
Done